### ISSUE TITLE
Initial Dynamic Behavior for Card Containers

### ISSUE THREAD

**Issue by LakeaTrenae** [tlg-24-05-visa-sde-2/24-05-Javascript-project-Team--2#4]
[Page to clone](https://www.figma.com/design/i9JIfSsRBLzSTdR7AG7EPq/Figma-basics?node-id=0-139&t=7ajQWaUKgur8uI4G-0)

- [ ] Use user input values that are already stored in variables to create a [bootstrap card](https://getbootstrap.com/docs/5.0/components/card/) for each grocery store item.

- [ ]  Append new cards inside of #cardContainers

**Comment by Bi-nary-Genius** [comment]
## Initial Dynamic Behavior for Card Containers

- [https://getbootstrap.com/docs/4.5/getting-started/introduction/](url)
- [https://unsplash.com/documentation](url)

This project dynamically creates and displays Bootstrap card containers based on user input and API data.

### Features:
- **Search Functionality**: Users can type keywords to search for images.
- **API Integration**: Fetch images from an API based on the search query.
- **Dynamic Cards**: Display images in Bootstrap card containers.
- **Responsive Layout**: Cards are responsive and adapt to different screen sizes.
- **Interactive Elements**: Click on a card to view the image in full size.
- **Error Handling**: Display a message if no images are found or if an error occurs.

### Instructions:
1. **Setup**: Initialize the project with the required dependencies (Bootstrap, jQuery, etc.).
2. **Search Input**: Create a search input field and button for user queries.
3. **Fetch Data**: Implement logic to fetch image data from an API based on the user’s search query.
4. **Display Cards**: Dynamically generate and display Bootstrap cards for each image in the response.
5. **Error Handling**: Show appropriate messages if the search fails or returns no results.
6. **Back to Top**: Include a button to scroll back to the top of the page.
7. **Responsive Design**: Ensure the layout is responsive and looks good on all devices.

### Functionalities TBD:

1. **Static Data**:
   - Use predefined data to populate Bootstrap card containers without API calls.

2. **API Integration**:
   - Fetch and display data from an external API based on user input.

These steps outline the process of implementing initial dynamic behavior for the card containers in the project.

**Comment by Bi-nary-Genius** [comment]
First try bootstrap and if time permits then work on the API integration by Wednesday.

**Comment by Bi-nary-Genius** [comment]
In the final draft of our project, we did not incorporate the bootstrap card, but instead created div elements to act as a container for the image results. A class named search-result was added to each container to be styled with CSS. Lastly, the img element was created and appended for the image and an "a" element for the hyperlink within each container (which in turn sets its href attribute to the url of the images found on Unsplash.
